packages/history-service/src/history-service-v1.ts

```diff
@@ -2,13 +2,10 @@ import * as history from 'history';
 import {RootLocationTransformer} from '.';
 import {ConsumerHistory} from './base-history';
 import {BrowserHistory} from './browser-history';
-import {MemoryHistory} from './memory-history';
 import {RootHistories} from './root-histories';
 
 export interface HistoryServiceV1 {
-  readonly rootLocation?: history.Location;
   createBrowserHistory(): history.History;
-  createMemoryHistory(): history.MemoryHistory;
 }
 
 export class HistoryService implements HistoryServiceV1 {
```

Comment:
i think this name does not scale in future :)

Reply:
fixed by refactoring to a `HistoryServiceV1Binding` class